canonical-webteam/global-nav
# Add basic responsive functionality

## Done
Added basic responsive functionality by styling the global nav to the bottom of the page on small screens.

## QA
- Pull down code
- Install by running `npm i` 
- Run `gulp test` and see there are no errors
- Then build using `gulp build`
- Open the index.html page in your browser
- Shrink the screen down to < 768px and see that the global nav jumps to the bottom and becomes an accordion. Like on http://www.ubuntu.com

## Details
Fixes https://github.com/canonical-webteam/global-nav/issues/14 https://github.com/canonical-webteam/global-nav/issues/2 https://github.com/canonical-webteam/global-nav/issues/1
